indra/llimage/llimagebmp.cpp

```diff
@@ -334,7 +330,7 @@ bool LLImageBMP::updateData()
 
 bool LLImageBMP::decode(LLImageRaw* raw_image, F32 decode_time)
 {
-	llassert_always(raw_image);
+	llassert_return_false(raw_image);
```

Comment:
This is a sign of problems with decoder, viewer should crash and report to bugsplat.